wiki/Rules/Explicit_Content/id.md

```diff
@@ -17,7 +12,7 @@ osu! mengizinkan pemakaian **konten audio** yang bersifat eksplisit (baik dari s
 
 **Di sisi lain, osu! tidak membenarkan penggunaan gambar maupun video yang bersifat senonoh - seluruh elemen visual yang ada tetap harus tunduk pada [Pertimbangan Konten Visual](/wiki/Rules/Visual_Content_Considerations).**
 
-Pengecualian di sini hanya diberikan pada elemen-elemen visual senonoh yang mengandung unsur estetika seni (*artistic use*) dalam nuansa dan konteks pemakaiannya.
+Pengecualian di sini hanya diberikan pada elemen-elemen visual senonoh yang mengandung unsur estetika seni (*artistic use*) dalam nuansa dan konteks pemakaiannya, seperti [storyboards](/wiki/Storyboards) atau konten video.
```

Comment:
`storyboards` should be `storyboard` as the Indonesian language doesn't use `s` at the end of a word to indicate a plural form